### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Properties_Blob Container Properties tab_ScreenReader: Voiceover announces 'Clickable' for all the non-interactive text edit fields associated with 'Properties' tab while using voiceover tab keys which is incorrect.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5767]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#Voiceover;#AINotInScope;#CEACCApproved;#A11yCAIOH;#Closed;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voice over 

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Navigate to any of the file inside "Blob Containers"
6. Tab to 'properties' tab.
7. Turn on voiceover and observe.

## Actual:
Voiceover announces 'Clickable' for all the non-interactive text edit fields associated with 'Properties' tab while using voiceover tab keys which is incorrect.

## Expected:
Voiceover should not announce 'Clickable' for the non-interactive text edit fields associated with 'Properties' tab while using voiceover tab keys.

## User Impact:
Screen reader users will believe that they will be able to make changes to the text edit field while in reality they won't be able to do so.

## Attachment:
![12_All the disabled](https://user-images.githubusercontent.com/98393966/169970086-9a2b458a-5e9c-417b-a401-ea3ccfe08b8a.gif)

**Comment by JasonYeMSFT** [comment]
Voice Over announces all read-only text boxes as "clickable" which isn't something we should be fighting against. Besides, we do have custom click handlers on those elements so they are indeed clickable anyway. I don't think we should do make any changes to these elements.

**Comment by bbarik06** [comment]
This is an expected behavior and thus, closing the bug.
